messaging/src/test/java/org/axonframework/configuration/LifecycleHandlerInspectorTest.java

```diff
@@ -46,80 +46,81 @@ class LifecycleHandlerInspectorTest {
     private static final int TEST_PHASE = 1;
 
     @Test
-    void nothingIsRegisteredForNullComponent(@Mock LifecycleSupportingConfiguration configuration) {
-        LifecycleHandlerInspector.registerLifecycleHandlers(configuration, null);
+    void nullComponentIsRejected(@Mock LifecycleRegistry operator) {
+        //noinspection DataFlowIssue
+        assertThrows(NullPointerException.class, () -> LifecycleHandlerInspector.registerLifecycleHandlers(operator, null));
 
-        verifyNoInteractions(configuration);
+        verifyNoInteractions(operator);
```

Comment:
Nit: I think we should rename all references of `operator` for `registry` or `lifecycleRegistry`.